test/e2e/common.go

```diff
@@ -39,6 +40,24 @@ func LogFromFile(logFile string) {
 	Logf(string(data))
 }
 
+func logTable(title string, columnNames []string, rows [][]string) {
+	getRowFormatted := func(row []string) string {
+		rowFormatted := row[0]
+		for i := 1; i < len(row); i++ {
+			rowFormatted = fmt.Sprintf("%s\t%s", rowFormatted, row[i])
+		}
+		return rowFormatted
+	}
+	w := tabwriter.NewWriter(GinkgoWriter, 0, 0, 2, ' ', tabwriter.TabIndent)
```

Comment:
A comment to explain this would be good. From the [docs](https://pkg.go.dev/text/tabwriter#Writer.Init) I understand that this means 0 minwidth, 0 tabwidth, 2 padding and one space used as padchar. But why should tabwidth be 0? I would expect that we set it to maybe 2 or 4.

Reply:
I think because I am not using tabs for padding it is meaningless to set any value there that what example in docs also did

Reply:
Ah that makes sense! I missed the second example... Never mind then :slightly_smiling_face: